scopes/preview/preview/env-preview-template.task.ts

```diff
@@ -138,33 +138,31 @@ export class EnvPreviewTemplateTask implements BuildTask {
   ): Promise<Target | undefined> {
     const env = envDef.env;
     const envPreviewConfig = this.preview.getEnvPreviewConfig(envDef.env);
-    const isSplitComponentBundle = envPreviewConfig.splitComponentBundle ?? false;
-    const peers = await this.dependencyResolver.getPeerDependenciesListFromEnv(env);
-    // console.log('envid22', env.__getDescriptor(), 'peers', peers)
 
+    const peers = await this.dependencyResolver.getPeerDependenciesListFromEnv(env);
     // const module = await this.getPreviewModule(envComponent);
     // const entries = Object.keys(module).map((key) => module.exposes[key]);
     const capsule = context.capsuleNetwork.graphCapsules.getCapsule(envComponent.id);
     if (!capsule) throw new Error('no capsule found');
     // Passing here the env itself to make sure it's preview runtime will be part of the preview root file
     // that's needed to make sure the providers register there are running correctly
     const previewRoot = await this.preview.writePreviewRuntime(context, [envComponent.id.toString()]);
-    const previewModules = await this.getPreviewModules(envDef);
-    // const templatesFile = previewModules.map((template) => {
-    //   return this.preview.writeLink(template.name, ComponentMap.create([]), template.path, capsule.path);
-    // });
+    const entries = await this.generateEntries({
+      envDef,
+      splitComponentBundle: envPreviewConfig.splitComponentBundle,
```

Comment:
in many cases this is undefined, that's why it's with `?? false`

Reply:
I set it during `generateEntries`. would you prefer I'd make it non-optional and set it here?